map/page.js

```diff
@@ -103,18 +103,14 @@ if (URLSearchParams && location.search && geocoder) {
 }
 
 async function geocode(address) {
-  return new Promise((resolve, reject) => {
-    try {
-      geocoder.geocode(address, (v) => {
-        v = v[0];
-        if (v) { v = v.center; }
-        resolve(v);
-      });
-    } catch (e) {
-      console.log("Problem:", e);
-      reject(e);
-    }
-  });
+  const results = await geocoder.geocode(address);
```

Comment:
This looks reasonable, but it seems like the original `try`-`catch` should be kept. Why is it no longer needed?

Reply:
I dropped it because it should just bubble out now due to using await, rather than needing explicit handling to conform it to the promise. But I can reinstate it with the logging out of caution, no problem.